sprints/sustainability/models.py

```diff
@@ -95,7 +108,8 @@ def calculate_budgets(self, start_date: datetime, end_date: datetime) -> None:
         # Calculate available budget for the next sprint.
         daily_budget: Dict[int, float] = {
             now.month: self.calculate_workday_budget(year, now.month, self.budgets[now.month - 1]),
-            now.month + 1: self.calculate_workday_budget(year, (now.month + 1) % 12, self.budgets[now.month % 12]),
+            (now.month + 1) % 12:
```

Comment:
This is weird, because I cannot reproduce it locally, but it's broken on prod. I tried debugging the prod and this should work.

Reply:
Not sure what you meant here but could it be that the key didn't `% 12` so it could go over 12?

Reply:
Yeah, that was the point. It shouldn't and I cannot reproduce it, but [that has actually happened](https://sentry.io/organizations/opencraft/issues/1391629621/?project=1463533&query=url%3Ahttp%3A%2F%2Fapi.sprints.opencraft.com%2Fsustainability%2Fdashboard%2F). We don't have too much time here, so I just patched this without time-consuming digging.